### PR TITLE
Update `sysrand` documentation

### PR DESCRIPTION
**Pull request by konsumlamm**

### CODE REVIEW

File: lib/std/sysrand.nim [hunk 1]
```diff
@@ -12,21 +12,21 @@
...
-## | Windows | `BCryptGenRandom`_ |
-## | Linux | `getrandom`_ |
-## | MacOSX | `getentropy`_ |
-## | IOS | `SecRandomCopyBytes`_ |
-## | OpenBSD | `getentropy openbsd`_ |
-## | FreeBSD | `getrandom freebsd`_ |
-## | JS(Web Browser) | `getRandomValues`_ |
-## | Nodejs | `randomFillSync`_ |
-## | Other Unix platforms | `/dev/urandom`_ |
+## | Targets              | Implementation        |
```
Comment (ghost) [inline lib/std/sysrand.nim:19]: any reason for aligning the table?  just curious

File: lib/std/sysrand.nim [hunk 2]
```diff
@@ -144,7 +144,7 @@ elif defined(windows):
     hAlgorithm: BCRYPT_ALG_HANDLE,
     pbBuffer: PUCHAR,
     cbBuffer: ULONG,
-    dwFlags: ULONG
+    dwFlags: ULONG,
```
Comment (ghost) [inline lib/std/sysrand.nim]: trailing comma?

File: lib/std/sysrand.nim (hunk 1 above)
Comment (konsumlamm) [reply lib/std/sysrand.nim:19]: Just so that it's easier to read.

File: lib/std/sysrand.nim (hunk 2 above)
Comment (konsumlamm) [reply lib/std/sysrand.nim]: Ye, for consistency with the other lines.

Comment (ringabout) [reply lib/std/sysrand.nim]: this should be discussed and change the nep1 or contributing guide.

**Comment by juancarlospaco** [comment]
I understand this is computationally more expensive than `random` module ones ?,
so maybe would be nice to add a line on the documentation explaining that...

File: lib/std/sysrand.nim (hunk 2 above)
Comment (Araq) [inline lib/std/sysrand.nim]:
```suggestion
    dwFlags: ULONG
```

Comment (Araq) [reply lib/std/sysrand.nim]: No trailing commas in parameter lists please.

Comment (timotheecour) [reply lib/std/sysrand.nim]: i agree with no commas in parameter lists; but IMO nep1 should recommend trailing commas for enums if there is 1 comma, eg:
```nim
type A1 = enum # good
  k1
  k2
type A2 = enum # good
  k1,
  k2,
type A3 = enum # bad
  k1,
  k2
```
(shorter diffs when adding an enum, more consistent, etc)

**Pull request merged**